common/fs/src/cache/mod.rs

```diff
@@ -1896,7 +1901,7 @@ mod tests {
 
     // Deletes a directory
     #[tokio::test]
-    #[cfg(unix)]
+    #[cfg(target_os = "linux")]
     async fn filesystem_delete_filled_dir() -> io::Result<()> {
         let tempdir = TempDir::new()?;
```

Comment:
why disabled on mac?

Reply:
There was kind of a cut bait and run strategy on some of these tests - Timing and directory creation issues that were found to be on the test side not code side.